src/demos/datepicker/fuzzy-datepicker-demo.component.html

```diff
@@ -0,0 +1,133 @@
+<h2
+  class="sky-section-heading"
+>
+  Standard fuzzy datepicker
+</h2>
+
+<div
+  style="max-width: 50%"
+>
+  <sky-datepicker>
+    <input
+      skyFuzzyDatepickerInput
+      [futureDisabled]="futureDisabled"
+      [dateFormat]="dateFormat"
+      [disabled]="disabled"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+      [(ngModel)]="selectedDate"
+      #date="ngModel"
+    />
+  </sky-datepicker>
+
+  <p *ngIf="date.errors && date.errors.skyFuzzyDate && (date.dirty || date.touched)"
+    class="sky-error-label"
+  >
+    <span *ngIf="date.errors.skyFuzzyDate?.invalid">
+      Please enter a valid date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.yearRequired">
+      Please select a year.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.futureDisabled">
+      Date cannot be in the future.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.maxDate">
+      The current date is beyond the maximum allowed date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.minDate">
+      The current date is before the minimum allowed date.
+    </span>
+  </p>
+</div>
+
+<p>
+  Selected fuzzy date is {{ selectedDateForDisplay }}
+</p>
+
+<h2
+  class="sky-section-heading"
+>
+  Reactive form fuzzy datepicker
+</h2>
+
+<form
+  novalidate
+  style="max-width: 50%"
+  [formGroup]="reactiveForm"
+>
+  <sky-datepicker>
+    <input
+      formControlName="selectedDate"
+      skyFuzzyDatepickerInput
+      [dateFormat]="dateFormat"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+    />
+  </sky-datepicker>
+</form>
+
+<p>
+  Reactive form selected fuzzy date is {{ reactiveFormSelectedDateForDisplay }}
+</p>
+
+<p *ngIf="reactiveDate.errors && (reactiveDate.dirty || reactiveDate.touched)"
+  class="sky-error-label"
+>
+  <span *ngIf="reactiveDate.errors.skyFuzzyDate?.invalid">
+    Please enter a valid date.
+  </span>
+  <span *ngIf="reactiveDate.errors.skyFuzzyDate?.yearRequired">
+    Please select a year.
+  </span>
+  <span *ngIf="reactiveDate.errors.skyFuzzyDate?.futureDisabled">
+    Date cannot be in the future.
+  </span>
+  <span *ngIf="reactiveDate.errors.skyFuzzyDate?.maxDate">
+    The current date is beyond the maximum allowed date.
+  </span>
+  <span *ngIf="reactiveDate.errors.skyFuzzyDate?.minDate">
+    The current date is before the minimum allowed date.
```

Comment:
See notes above, but what do you think about changing this error message to `This date is earlier than allowed. Please select a date after [min date].`?

Reply:
Looks good, but same question about formatting the date.

Reply:
Date formatted.